bundles/action/org.openhab.action.telegram/README.md

```diff
@@ -15,7 +15,7 @@ As described in the Telegram Bot API, this is the manual procedure needed in ord
 
 - Open a new chat with your new Bot and post a message on the chat
 - Open a browser and invoke `https://api.telegram.org/bot<token>/getUpdates` (where `<token>` is the authentication token previously obtained)
-- Look at the JSON result and write down the value of `result[0].message.chat.id`. That is the chatId.
+- Look at the JSON result and write down the value of `result[0].message.chat.id`. That is the chatId. Note that Telegram group chat chatIds are prefixed with a dash "-" that must be included in the config file. (eg: bot1.chatId: -22334455)

```

Comment:
Nice work @vzorglub!  You are very close.  Just one thing please, and it is a very, very small thing.  The formatting of the parenthetical expression should be like this... "in the config file (e.g. bot1.chatId: -22334455). 
With that change, it looks good to me.  Thank you!!